cmd/allocator/metrics.go

```diff
@@ -42,6 +42,7 @@ const (
 	apiServerSustainedQPSFlag        = "api-server-qps"
 	apiServerBurstQPSFlag            = "api-server-qps-burst"
 	logLevelFlag                     = "log-level"
+	allocationBatchWaitTime          = "allocation-batch-wait-time"
```

Comment:
So this got me curious - since the value doesn't change (at least in a single install), why export is as a metric?

Reply:
OIC, this is just bad file naming. We should clean that up at some point.